.github/workflows/build.yml

```diff
@@ -80,19 +80,36 @@ jobs:
 
   deploy_to_clojars:
     runs-on: ubuntu-latest
-    needs: [lint_check, run_tests_with_kafka_1, run_tests_with_kafka_2, calculate_coverage]
+    needs:
+      [
+        lint_check,
+        run_tests_with_kafka_1,
+        run_tests_with_kafka_2,
+        calculate_coverage,
+      ]
     if: ${{ startsWith(github.ref, 'refs/tags/') }}
     env:
       CLOJARS_USERNAME: ${{ secrets.CLOJARS_USERNAME }}
       CLOJARS_PASSWORD: ${{ secrets.CLOJARS_PASSWORD }}
     steps:
       - uses: actions/setup-java@v1
         with:
-          java-version: '1.8'
+          java-version: "1.8"
       - uses: DeLaGuardo/setup-clojure@master
         with:
-          lein: '2.8.1'
+          lein: "2.8.1"
       - uses: actions/checkout@v2
 
       - name: Deploy to Clojars
         run: lein deploy clojars
+
+  send_update_event:
+    runs-on: ubuntu-latest
+    needs: [deploy_to_clojars]
+    if: ${{ startsWith(github.ref, 'refs/tags/') }}
+    steps:
+      - uses: peter-evans/repository-dispatch@v1
+        with:
+          token: ${{secrets.GH_TOKEN}}
+          repository: shubhang93/ziggurat-web #TODO change to gojek/ziggurat-web
```

Comment:
Why not change it to gojek/ziggurat-web right away?

Reply:
we wanted to test it with our forks, once we acquire the gojek credentialsm, we will go ahead and change it